test/relayer_test.go

```diff
@@ -18,6 +19,10 @@ func getTestChainFactory() ibc.ChainFactory {
 	)
 }
 
+func TestRelayerByRelayerTest(t *testing.T) {
```

Comment:
This name is bad, but it was easy to use for filtering.

The structure of this test is identical to how a separate project, e.g. cosmos/relayer, could import and run the tests locally.

The intent is that the other project would pick a builtin chain factory and would provide its own relayer factory. The in-project tests are not intended to replace CI running a full suite of tests against many chains; they are merely a developer convenience.

I have changes to cosmos/relayer on my workstation that follow this pattern, but I'm not ready to push those changes until we update ibc-test-framework to allow it.